test/regression/2019-08-02-missing-switch-default.ll

```diff
@@ -0,0 +1,51 @@
+; REQUIRES: geq-llvm-3.8
+; RUN: rm -rf %t.klee-out
+; RUN: llvm-as -f %s -o %t.bc
+; RUN: %klee --switch-type=internal --output-dir=%t.klee-out %t.bc
+; RUN: FileCheck --input-file=%t.klee-out/info %s
+; CHECK: KLEE: done: completed paths = 3
+
+target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
```

Comment:
Can you try to remove the datalayout information - that should not be necessary if I'm not mistaken.